src/Raven.Studio/wwwroot/App/views/database/patch/patch.html

```diff
@@ -262,3 +272,42 @@ <h2>Select patch</h2>
         </div>
     </div>
 </script>
+
+<script type="text/html" id="settings-template">
+    <div class="padding padding-xs">
+        <div class="margin-left margin-left-lg">
+            <strong><h4>Patching behavior for stale indexes:</h4></strong>
+            <div class="radio">
+                <input id="allowPatchingStaleIndex" type="radio" data-bind="checked: staleIndexBehavior" name="staleBehavior" value="patchStale">
+                <label for="allowPatchingStaleIndex">Patch immediately</label>
+            </div>
+            <div class="radio margin-bottom margin-bottom-sm">
+                <input id="timeToWaitForNonStale" type="radio" data-bind="checked: staleIndexBehavior" name="staleBehavior" value="timeoutDefined">
+                <label for="timeToWaitForNonStale">Define timeout to wait for index to become non-stale</label>
+                <div class="margin-left margin-left-lg" data-bind="collapse: staleIndexBehavior() === 'timeoutDefined'">
+                    <input type="text" class="form-control" data-bind="durationPicker: staleTimeout, durationPickerOptions: { showDays: false, showSeconds: true }" />
+                </div>
+                <div class="margin-left margin-left-lg margin-top margin-top-sm" data-bind="visible: staleIndexBehavior() === 'timeoutDefined'">
+                    <small><i class="icon-warning"></i>  An exception is thrown when timeout has passed</small>
+                </div>
+            </div>
+        </div>
+    </div>
+    <hr class="margin-left margin-left-lg margin-right margin-right-lg">
+    <div class="padding padding-xs">
```

Comment:
can we use padding here only, remove padding-xs, and then remove inner div with margin-left margin-left-lg to avid so many padding inside margin inside padding inside margin ;-)

Reply:
Played w/ this some more and removed some excess classes.
Now looking like this:

![image](https://user-images.githubusercontent.com/17376974/74879975-f92bdd00-5372-11ea-9163-5baa0a628bc0.png)
![image](https://user-images.githubusercontent.com/17376974/74879988-021cae80-5373-11ea-8261-854249c72cd6.png)